### DISCUSSION TITLE
Daily Standup (Tuesday, May 31th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#81] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
 ```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked with @CarlosCRG19 on Signup

**What are your priorities for the next 24 hours?**:
 - Keep working on it

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Merged some PR's
 - Send #78 #79 

**What are your priorities for the next 24 hours?**:
 - Work on signup flow

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto @BrandonMagana for their hard work on the back. They both prepared PR's

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Finished Skill's controllers and routes
 -  Implemented @antonieto suggestions on my PR

**What are your priorities for the next 24 hours?**:
 -  Disccuss with @antonieto the new endpoint and how to implement it

**Blockers**:
 - None

**Shoutouts**:
- @CarlosCRG19  for helping me with git

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Kept working on the search aspirant form

**What are your priorities for the next 24 hours?**:
 - Enable auth 

**Blockers**:
 - Auth

**Shoutouts**:
 - @DavBot02

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Start aspirant-skills relation 

**What are your priorities for the next 24 hours?**:
 - Finish aspirant-skills relation 
 - Filter aspirants by skill

**Blockers**:
 - None

**Shoutouts**:
 - @everyone
